tests/test_meta/test_grouped_model.py

```diff
@@ -16,9 +16,7 @@
     estimator_checks.check_estimators_nan_inf,
     estimator_checks.check_estimators_overwrite_params,
     estimator_checks.check_estimators_pickle,
-    estimator_checks.check_fit2d_predict1d,
     estimator_checks.check_fit2d_1sample,
-    estimator_checks.check_fit1d,
```

Comment:
if you remove checks thats perfectly fine, but please add a comment indicating why it has been removed. this makes it easier to maintain.